cluster-service/deploy/templates/deployment.yaml

```
@@ -136,6 +136,13 @@ spec:
         - name: azure-operators-managed-identities-config
           mountPath: /configs/azure-operators-managed-identities-config.yaml
           subPath: azure-operators-managed-identities-config.yaml
+        {{- if .Values.tracing.enabled }}
```

```suggestion
        {{- if ne .Values.Tracing.Address "" }}
```

Maybe we want to replace the `enabled` field with a check for an empty address?